ras_party/views/account_view.py

```diff
@@ -164,14 +164,41 @@ def post_pending_survey_respondent():
     return make_response(jsonify(response), 201)
 
 
-@account_view.route("/respondents/<respondent_id>/password-verification-tokens", methods=["POST"])
-def post_password_verification_tokens(respondent_id):
+@account_view.route("/respondents/<respondent_id>/password-verification-token", methods=["GET"])
+def get_password_verification_token(respondent_id):
+    token = account_controller.get_respondent_password_token(respondent_id)
+    return make_response(jsonify({"token": token}), 200)
+
+
+@account_view.route("/respondents/<respondent_id>/password-verification-token", methods=["POST"])
+def post_password_verification_token(respondent_id):
     payload = request.get_json()
     account_controller.add_respondent_password_token(respondent_id, payload["token"])
     return make_response(jsonify({"message": "Successfully added token"}), 200)
 
 
-@account_view.route("/respondents/<respondent_id>/password-verification-tokens/<token>", methods=["DELETE"])
-def delete_password_verification_tokens(respondent_id, token):
+@account_view.route("/respondents/<respondent_id>/password-verification-token/<token>", methods=["DELETE"])
+def delete_password_verification_token(respondent_id, token):
     account_controller.delete_respondent_password_token(respondent_id, token)
     return make_response(jsonify({"message": "Successfully removed token"}), 200)
+
+
+@account_view.route("/respondents/<respondent_id>/password-reset-counter", methods=["GET"])
+def get_password_reset_counter(respondent_id):
+    counter = account_controller.get_password_counter(respondent_id)
+    return make_response(jsonify({"counter": counter}), 200)
```

Comment:
for all of these simple endpoints that have 200 return values , doing:
```python
return jsonify(<insert dict here>)
```
will suffice.

I think (but you'll have to check... maybe in this repo or collection instrument?) you can do the same for a non-200 value, where it looks something like:
```python
return jsonify(<insert dict here>), 400
```